exercises/concept/freelancer-rates/.docs/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+There are two different types of numbers in JavaScript:
+
+- `number`: a numeric data type in the double-precision 64-bit floating point format (IEEE 754).
+  Examples are `-6`, `-2.4`, `0`, `0.1`, `1`, `3.14`, `16.984025`, `25`, `976`, `1024.0` and `500000`.
+- `bigint`: a numeric data type that can represent _integers_ in the arbitrary precision format.
+  Examples are `-12n`, `0n`, `4n`, and `9007199254740991n`.
```

Comment:
In theory, this could be omitted because the exercise doesn't use it and thus the student does not have to know about it upfront. That said, I can see why you'd want to list them both here.